### ISSUE TITLE
Please add   the CRX file here in this repository.

### ISSUE THREAD

**Issue by korenevskiy** [Rob--W/crxviewer#125]
I can't download extensions from the Chrome Store. It seems to me that those who use this extension, some of them also cannot work with the Chrome Store. Please add a direct link in this repository to the CRX file. Upload this file here!

**Comment by Rob--W** [comment]
Some organizations have disabled access to the Chrome Web Store when you're logged in to a Google account. Assuming that this is the case, and not an enterprise policy-enforced restriction blocking installation at the browser level, then you can try to install the extension if you have a crx file.

To get the crx file for crxviewer, you could try to use the online demo of crxviewer:

1. Visit https://robwu.nl/crxviewer/
2. In the "Extension ID" box, input jifpbeccnghkjeaalbbjmodiffmgedin.
3. The real location of the CRX file is displayed in the input box. You can try to download that directly by copying it and pasting it in your location bar. If that doesn't work, you can proceed:
4. Click on "Open in this viewer". The online demo will fetch the crx file (through my proxy, so it will not be affected by a network-level block if any)
5. When the viewer has been loaded, a "CRX" link appears in the upper-right corner. Click on this link to download the unmodified CRX file. NOTE: Do NOT click on "Download", because that will yield the zip file generated from the crx file.

TL;DR: Use https://robwu.nl/crxviewer/ to get the crx of crxviewer.

**Comment by korenevskiy** [comment]
People often want to download an extension as a file because the extension has already been removed from publication.
Otherwise, there is no point in saving it to the archive in such cases. Set a plan to add the ability to save the extension immediately from the Chrome folder. Without downloading from the Internet.

**Comment by Rob--W** [comment]
Are you requesting the ability to read the CRX from your filesystem? An extension cannot do that.
